primitives/src/vstaging/executor_params.rs

```diff
@@ -39,6 +39,10 @@ pub enum ExecutorParam {
 	/// Max. amount of memory the preparation worker is allowed to use during
 	/// pre-checking, in bytes
 	PrecheckingMaxMemory(u64),
+	/// PVF preparation timeouts, millisec
+	PvfPrepTimeout(PvfPrepTimeoutKind, u64),
```

Comment:
Does it make sense to annotate with `#[codec(index = X)]` to be explicit about backwards compatibility?